js/oop/Palindrome.js

```diff
@@ -2,16 +2,67 @@ import { parseString } from "../helper.js";
 
 export class Palindrome {
   /** @type {string} */
-  value;
+  word;
 
   /**
-   * @param {string} value
+   * @param {string} word
    */
-  constructor(value) {
-    this.value = parseString(value);
+  constructor(word) {
+    this.word = parseString(word);
+    this.arrayWord = this.word.split("");
   }
 
-  // write your code here
+  isUsingReverse() {
+    const reverseWord = this.arrayWord.reverse().join("");
+
+    return this.word === reverseWord;
+  }
+
+  isUsingLoop() {
+    let length = this.arrayWord.length;
+
+    for (let i = 0; i < length / 2; i++) {
+      if (this.arrayWord[i] !== this.arrayWord[length - 1 - i]) {
+        return false;
+      }
+    }
```

Comment:
I prefer to use "index" than just "i" as the name for this loop variable because it's much more distinctive to understand for the other.